docs/plugins/harbor.md

```diff
@@ -0,0 +1,93 @@
+# harbor Plugin
+
+This plugin installs [harbor](https://goharbor.io/) in an existing Kubernetes cluster using the Helm chart.
+
+## Usage
+
+##### External storage
+
+- Postgresql: Set the `database.type` to `external` and fill the information in `database.external` section.
+- Redis: Set the `redis.type` to `external` and fill the information in `redis.external` section.
+
+##### Disk storage
+
+Please be sure to change the `storageClass` in the options of the config to an existing StorageClass. You can refer to this [document](https://github.com/goharbor/harbor-helm#configure-how-to-persist-data)
```

Comment:
Here maybe we can add some more help for developers who want to test it locally. See other plugins (like Jenkins) doc for more info, like how to create a hostPath storage class, etc.

Reply:
Missing comma at the end of the sentence. Check all sentences, please. [Grammarly](https://app.grammarly.com/) is recommended for all English documentation work.